### ISSUE TITLE
after writing image with win32 image writer sd card dead

### ISSUE THREAD

**Issue by noobline** [resin-io/etcher#2396]
hello , 

I read some articles here but didnt find the answer for my problem 

so..

the issue is, after writing img to sd card it worked for a while
now not possible to do anything with

please advise what should be done

sd card is for 32gb but shows just 128mb
![image](https://user-images.githubusercontent.com/40339504/41509796-99237158-7259-11e8-93bf-a5cf636c81a6.png)

as you can see from screenshot there is no parts that windows cant see or smth.
I tried the cmd commands with a diskpart and etc. nothing work, alsoo deep formatting didnt help
SD car formatter also telling some bullshit
Basically sometimes I got message like it is locked from writing but its not true

tell if this card still alive or need be in trash?)

**Comment by lurch** [comment]
It might be a 'fake' SD card that you bought thinking it was 32GB, but in reality it's only 128MB?
See https://sosfakeflash.wordpress.com/ for more details. You can use H2TestW to be sure...

**Comment by noobline** [comment]
Noo, it was ok SD card
I wrote on it couple times img with win32Img writer...if im correct with the name of program...
It suppose to be for Raspberry PI

SO it was working and it is not...at all

**Comment by lurch** [comment]
I don't know if win32 image writer does any verification? It's possible that the card was happily "accepting" the data, but not actually storing it? As I said above, you need to test with H2TestW to be sure.

And even if win32 image writer did "break" your SD card, I guess that's not really anything to do with Etcher?

**Comment by jhermsmeier** [comment]
@noobline did you use win32diskimager, or Etcher to flash your SD-card?

**Comment by txoof** [comment]
I have a similar issue with a stack of SD cards and USB drives. I've used Etcher to write several Raspberry PI images for a variety of projects. Wanting to reuse these drives, I've attempted to remove these partitions and found that the drive/sd card is stuck in a read-only format. At first I suspected the problem was with the cheap drives I was using, but that I now have a stack of both generic drives and several genuine SanDisk SD Cards that are junk, I believe the problem may be deeper though I'm not entirely sure the issue is with Etcher. Is it possible that something in the PI images (mostly Raspbian) could be setting a read-only bit somewhere on the drive? I don't even know if this is possible, but I'm totally at a loss here.

I've attempted the following:
**OS X High Sierra:**
* Use Tuxera SD Card Formatter in both Quick and Overwrite modes
`SDHC formatting was successfully completed.
Volume information:
File system: FAT32
Capacity: 15.83 GB (15,833,497,600 bytes)
Free space: 15.83 GB (15,833,464,832 bytes)
Cluster size: 32,768 bytes
Volume label: NONE
$ diskutil list
/dev/disk3 (external, physical):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:     FDisk_partition_scheme                        *15.8 GB    disk3
   1:             Windows_FAT_16 boot                    43.8 MB    disk3s1
   2:                      Linux                         15.8 GB    disk3s2`
   * RESULT: sd card is unusable 
* Use disk utility to unmount (eject all visible partitions) and erase all partitions GUI fails:
`Unmounting disk
Creating the partition map
Waiting for partitions to activate
Formatting disk3s1 as MS-DOS (FAT) with name UNTITLED
This operation requires an unmounted disk.
Operation failed…
`
  * RESULT: drive is still unusable
* diskutil on the command line as per the recommendations on the wiki fails to erase the partitions
`$ diskutil list
/dev/disk3 (external, physical):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:     FDisk_partition_scheme                        *15.8 GB    disk3
   1:             Windows_FAT_16 boot                    43.8 MB    disk3s1
   2:                      Linux                         15.8 GB    disk3s2
$ sudo diskutil eraseDisk free UNTITLED /dev/disk3
Started erase on disk3
Unmounting disk
Creating the partition map
Waiting for partitions to activate
Finished erase on disk3
$ sudo diskutil eraseDisk free UNTITLED /dev/disk3
Started erase on disk3
Unmounting disk
Creating the partition map
Waiting for partitions to activate
Finished erase on disk3
$ diskutil list
/dev/disk3 (external, physical):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:     FDisk_partition_scheme                        *15.8 GB    disk3
   1:             Windows_FAT_16 boot                    43.8 MB    disk3s1
   2:                      Linux                         15.8 GB    disk3s2
`
   * RESULT: disk is still unusable
* Using dd on the command line as suggested by the wiki fails:
`$ sudo dd if=/dev/zero of=/dev/disk3 bs=512 count=1 conv=notrunc
1+0 records in
1+0 records out
512 bytes transferred in 0.000827 secs (619050 bytes/sec)
$ diskutil list
/dev/disk3 (external, physical):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:     FDisk_partition_scheme                        *15.8 GB    disk3
   1:             Windows_FAT_16 boot                    43.8 MB    disk3s1
   2:                      Linux                         15.8 GB    disk3s2`
   * RESULT: disk is unusable


**Ubuntu 18.04 LTS**
* Using dd:

`$ sudo dd if=/dev/zero of=/dev/sde bs=512 count=1 conv=notrunc
1+0 records in
1+0 records out
512 bytes copied, 0.00135433 s, 378 kB/s
$ fdisk /dev/sde
Disk /dev/sde: 14.8 GiB, 15841886208 bytes, 30941184 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0xe60c8050
Device     Boot Start      End  Sectors  Size Id Type
/dev/sde1        8192    93814    85623 41.8M  e W95 FAT16 (LBA)
/dev/sde2       94208 30941183 30846976 14.7G 83 Linux`
   * RESULT: unusable
* using fdisk and cfdisk yield same result

**Comment by toothbrush7777777** [comment]
@noobline, @txoof Did you check that the write protect switch at the side of the SD card [adapter] is disabled (towards the top of the SD card, near the metal contacts)?

**Comment by txoof** [comment]
Yes. I definitely checked this. When working with the micro-sd cards I used three different fuul size SD card adapters (switch up and down for good measure) with two Mac computers with SD card slots, an external SD card reader in both OS X and and a Linux box. 

The media is well and true euchered; the problem is not the switch.
